package.json

```diff
@@ -32,6 +32,27 @@
     "ext": "js",
     "exec": "babel-node ./src/server/"
   },
+  "eslintConfig": {
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended"
+    ],
+    "rules": {    
+      "react/react-in-jsx-scope": "off",
+      "semi" : ["warn", "always", { "omitLastInOneLineBlock" : true }]
+    },
+    "globals": {    
+      "React": "writable"
+    },
+    "parserOptions": { 
+      "ecmaVersion": 6,
```

Comment:
Enabled es6 checks.

Reply:
If we extend `react-app`, we automatically get `ecmaVersion: 2018` and these other `parserOptions`.

Reply:
Fixed in the new version of the file.